README.rst

```diff
@@ -375,6 +375,55 @@ Usage exmaple:
     >>> user.organizations_dict.keys()
     ... dict_keys(['20135c30-d486-4d68-993f-322b8acb51c4'])
 
+Organization membership permission_classes
+------------------------------------------
+
+The custom rest_framework permission classes ``IsOrganizationMember``,
+``IsOrganizationManager`` and ``IsOrganizationOwner`` can be used in the
+API to ensure that the request user is in the same organization as requested
+object and is organization member, manager or owner respectively.
+
+Usage exmaple:
+
+.. code-block:: python
+
+    from openwisp_users.multitenancy import IsOrganizationManager
+
+    class MyApiView:
+        permission_classes = (IsOrganizationMember,)
+
+        def get(self, request, *args, **kwargs):
```

Comment:
if the view extends APIView or any other DRF view, this should not be needed, hence we can provide the example pointing to the standard usage.